app/src/main/AndroidManifest.xml

```diff
@@ -11,10 +11,19 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.NFC" />
 
+    <uses-permission android:name="android.permission.BLUETOOTH"/>
+    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN"/>
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
+
+
     <uses-feature
         android:name="android.hardware.camera.autofocus"
         android:required="false" />
 
+    <uses-feature
+        android:name="android.hardware.bluetooth_le"
```

Comment:
What if users don't have BLE and a Ledger Nano? Shouldn't they still be able to use our app without BLE?

Reply:
definitely. This line is to be removed